### PR TITLE
switch to GHA

### PR DESCRIPTION
**Pull request by almindor**
Switch to gibhub Actions

### CODE REVIEW

**Comment by almindor** [comment]
@Disasm we need this to unblock broken travis builds

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      - name: Run CI script for riscv32imac-unknown-none-elf under ${{ matrix.rust }}
+        run: TARGET=riscv32imac-unknown-none-elf cargo check
+      - name: Run CI script for riscv64imac-unknown-none-elf under ${{ matrix.rust }}
+        run: TARGET=riscv64imac-unknown-none-elf cargo check
+      - name: Run CI script for riscv64gc-unknown-none-elf under ${{ matrix.rust }}
+        run: TARGET=riscv64gc-unknown-none-elf cargo check
+      - name: Check blobs
+        run: ./check-blobs.sh
+
+  # On macOS and Windows, we at least make sure that all examples build and link.
```
Comment (Disasm) [inline .github/workflows/ci.yaml:44]:
```suggestion
  # On macOS and Windows, we at least make sure that the crate builds.
```

**Comment by Disasm** [comment]
Could you also modify bors.toml like this (with correct msrv)? https://github.com/rust-embedded/cortex-m-rt/blob/master/.github/bors.toml#L4-L10

**Comment by almindor** [comment]
> Could you also modify bors.toml like this (with correct msrv)? https://github.com/rust-embedded/cortex-m-rt/blob/master/.github/bors.toml#L4-L10

@Disasm should be good to go now

**Review by Disasm — APPROVED** [review]
Thank you a lot!
bors r+

**Comment by Disasm** [comment]
bors cancel

**Pull request closed without merging**